x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/create.ts

```diff
@@ -261,18 +261,11 @@ export default function createAlertTests({ getService }: FtrProviderContext) {
           switch (scenario.id) {
             case 'no_kibana_privileges at space1':
             case 'space_1_all at space2':
-              expect(response.statusCode).to.eql(403);
-              expect(response.body).to.eql({
-                error: 'Forbidden',
-                message: getUnauthorizedErrorMessage('create', 'test.noop', 'alerts'),
-                statusCode: 403,
-              });
-              break;
             case 'global_read at space1':
               expect(response.statusCode).to.eql(403);
               expect(response.body).to.eql({
                 error: 'Forbidden',
-                message: getUnauthorizedErrorMessage('create', 'test.noop', 'alertsFixture'),
+                message: getUnauthorizedErrorMessage('create', 'test.noop', 'alerts'),
```

Comment:
The rule in the tests is created with the `alerts` consumer. The `alertFixture` is the feature ID and is not relevant anymore. This is the case for all the integration tests. The status code and the permission did not change. Still a 403 is thrown but with a small change in the message.